src/testdir/test_system.vim

```diff
@@ -1,25 +1,23 @@
 " Tests for system() and systemlist()
 
 function! Test_System()
-  if !executable('echo') || !executable('cat') || !executable('wc')
-    return
-  endif
-  let out = system('echo 123')
-  " On Windows we may get a trailing space.
-  if out != "123 \n"
-    call assert_equal("123\n", out)
+  if !has('win32')
+    call assert_equal("123\n", system('echo 123'))
+    call assert_equal(['123'], systemlist('echo 123'))
+    call assert_equal('123',   system('cat', '123'))
+    call assert_equal(['123'], systemlist('cat', '123'))
+    call assert_equal(["as\<NL>df"], systemlist('cat', ["as\<NL>df"]))
+  else
+    call assert_equal("123\n", system('cmd /c echo 123'))
```

Comment:
Why is `cmd /c` used here?
For someone who sets shell to bash.exe or something?